test/package.test.js

```diff
@@ -147,6 +147,16 @@ describe('package', function () {
         Expect(version).to.eql('1.2.3');
       });
     });
+
+    it('returns null if no version is specified', function () {
+      Package.getUserPackage.restore();
+      Sinon.stub(Package, 'getUserPackage').returns(Bluebird.resolve({ version: '' }));
+
+      return Package.calculateNewVersion()
+      .then(function (version) {
+        Expect(version).to.eql(null);
+      });
+    });
```

Comment:
Nit pick: can you add a new line after this `it` block? We usually pad our `it` blocks with new lines on the top and bottom.